shared/alembic/versions_opencore/alembic_2022_07_19_16_21_bfb007d3c340_tags.py

```diff
@@ -0,0 +1,44 @@
+"""tags
+
+Revision ID: bfb007d3c340
+Revises: 625370e1484b
+Create Date: 2022-07-19 16:21:55.940920
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import datetime
+
+# revision identifiers, used by Alembic.
+revision = 'bfb007d3c340'
+down_revision = '625370e1484b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('job_tag',
```

Comment:
MIght be a good idea to add indexes from the getgo to project_id and job id maybe?

Reply:
My understanding was that postgres would do that automatically for primary keys https://www.postgresql.org/docs/current/indexes-unique.html
"PostgreSQL automatically creates a unique index when a unique constraint or primary key is defined for a table. The index covers the columns that make up the primary key or unique constraint (a multicolumn index, if appropriate), and is the mechanism that enforces the constraint."

But not 100% sure if that covers your question given the triplicate primary keys